Client/Application/AppDelegate.swift

```diff
@@ -716,6 +720,7 @@ extension AppDelegate: UNUserNotificationCenterDelegate {
                 break
             case .readingList:
                 addToReadingList(response.notification)
+                UnifiedTelemetry.recordEvent(category: .action, method: .add, object: .readingListItem, value: .shareExtension)
```

Comment:
I don't think this code is actually used, this is supposed to be triggered with a long press on an incoming notiifcation, but I don't think that works.

The `.shareExtension` would be via the incoming queue I think. Or it would have to be measured in the actual extension.

Reply:
Ahhh.. you're right. I'm not sure how else to hook it then. We can try doing it from the actual extension code, but we haven't tested `telemetry-ios` in that environment yet.